.github/workflows/build.yml

```diff
@@ -63,13 +68,45 @@ jobs:
       run: |
         ./9_sycl_of_hell -install
 
+    - name: Run CMake examples
+      if: ${{ matrix.use_cmake }}
+      working-directory: 9_sycl_of_hell
+      run: |
+        source /opt/intel/oneapi/setvars.sh
+
+        ./9_sycl_of_hell -run 0_tiny_sycl_info
```

Comment:
Can we just do ` run-all` and do all the error checking / early exist inside the `.sh` scipt? Because if always a pain to edit the `build.yml` (and easiest to maintain the `.sh` than a crazy yml)